org.devmiyax.yabasanshiro.yml

```diff
@@ -0,0 +1,47 @@
+app-id: org.devmiyax.yabasanshiro
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: yabasanshiro
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # Needs to talk to the network:
+  - --share=network
+  # Needs to save files locally
+  - --filesystem=home
+  # OpenGL,USB controller
+  - --device=all
+  # Sound
+  - --socket=pulseaudio
+modules:
+  - name: boost
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap.sh --prefix=/app 
+      - ./b2 -j $FLATPAK_BUILDER_N_JOBS install
+    sources:
+      - type: archive
+        url: https://boostorg.jfrog.io/artifactory/main/release/1.78.0/source/boost_1_78_0.tar.bz2
+        sha256: 8681f175d4bdb26c52222665793eef08490d7758529330f98d3b29dd0735bccc
+
+  - name: yabasanshiro
+    buildsystem: simple
+    build-options:
+      build-args:
+       - --share=network
```

Comment:
The builds need to be performed offline. Is there anything fetched from network that you decided to specify it?

Reply:
Because I use ExternalProject_Add  which clones from the git repository at build time.

```cmake
ExternalProject_Add(
     Json
     PREFIX ${CMAKE_CURRENT_BINARY_DIR}/Json
     GIT_REPOSITORY https://github.com/nlohmann/json.git
```

>The builds need to be performed offline. 
Is this mandatory?

Reply:
yes